src/gui/css/loginScreen.css

```diff
@@ -0,0 +1,24 @@
+.AppName {
```

Comment:
is not a good practice to use uppercase letters in CSS, please change them all to kebab case:
`.AppName` --> `.app-name`

Reply:
okok, thanks (: